### PR TITLE
Initial delivery of patient-browser tests

### PR DESCRIPTION
**Pull request by rguderian**
Contains the Sort By and Page Scrolling Tests
Contains the patient loading script for ingesting patients with AddNLPInsights=true

Signed-off-by: Roger Guderian <roger.guderian@ibm.com>

### CODE REVIEW

File: ingest/src/test/java/ui/pageobjects/PatientList.java [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+	
+	private WebElement prevButton() {
+		return waitAndFindElement(By.className("fa-arrow-left"));
+	}
+	
+	private WebElement nextButton() {
+		return waitAndFindElement(By.className("fa-arrow-right"));
+	}
+	
+    private WebElement sortButton(String buttonName) {
```
Comment (ranum) [inline ingest/src/test/java/ui/pageobjects/PatientList.java:54]: nit: indentation?

File: ingest/src/test/java/ui/tests/PatientListPageTests.java [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+	}
+	
+	@Test
+	public void sortDescendingByIDTest() throws InterruptedException {
+
+		// click twice to perform the Descending sort
+		patientListPage.clickOnSortByID();			
+		patientListPage.clickOnSortByID();
+		
+        List<String> patientIDs = patientListPage.getPatientIDsOnPage();
```
Comment (ranum) [inline ingest/src/test/java/ui/tests/PatientListPageTests.java:70]: nit: There are many lines in this file that seem indented wrong, at least in the git tool.  Might not be an issue but it looks strange when doing the review.

**Review by ranum — APPROVED** [review]
LGTM

**Review by mcbride-p — COMMENTED** [review]
Eventhough all methods in PatientList.java are well named and code logic is pretty clear for each would be good to include explicit comment at start of each that explains purpose of method, input and output.

Ditto for each test case in PatientListPageTest.java.

Otherwise LGTM.

**Pull request merged**